### ISSUE TITLE
Fetcher store always returns `loading: false` despite ongoing Promise

### ISSUE THREAD

**Issue by jilio** [nanostores/query#12]
Hello,

I've been experiencing an issue with the fetcher store's loading state in the nanoquery package. Despite the fetcher returning a Promise that resolves after a delay, the loading state remains false throughout the duration of the Promise.

Here is a simplified code snippet that reproduces the issue:

```typescript
export const [createFetcherStore, createMutatorStore] = nanoquery({
  fetcher: (...keys: string[]) => {
    return new Promise((resolve) => {
      setTimeout(() => {
        resolve({data: 'some data'})
      }, 2000)
    })
  },
})
```

In this example, I would expect the `loading` state to be `true` during the 2-second delay before the Promise resolves. However, I'm consistently observing that loading remains `false`, despite the fact that the data arrives as expected after the 2 seconds.

I would appreciate any guidance on this issue. If there's any additional information or context that would help diagnose the issue, please let me know and I'll provide it promptly.

Thank you for your time and assistance.

**Comment by dkzlv** [comment]
@jilio Yes, I can confirm there's a bug with an initial state there. I'll have a close look there, thanks.

**Comment by jilio** [comment]
That is awesome! ✨ 
Thank you @dkzlv
